### DISCUSSION TITLE
Furniture Sit

### DISCUSSION THREAD

**Discussion by 00Hrtz** [TownyAdvanced/Towny#6150] [Q&A]
Hi,

I use the `Itemsadder` plugin which has a worldguard  `ia-furniture-sit`  flag.

How can i allow players to sit on custom furniture in plots?

**Comment by LlmDl** [comment]
While OP'd and holding a clay brick item in your hand, hit the furniture. Towny will tell you what that entity/block really is and I would need that information to help you further.
